app/controllers/api/v1/users/events_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Users::EventsController < ApplicationController
+  def index
+    render json: MyEventSerializer.new(User.find(params[:id]).events, { params: { user_id: params[:id] } })
```

Comment:
Nice use of passing the optional params in so that we can use them in the serializer!